TrackletAlgorithm/VMStubMEMemoryCM.h

```diff
@@ -71,9 +71,10 @@ class VMStubMECM : public VMStubMECMBase<VMSMEType>
   };
 
   typedef ap_uint<VMStubMECMBase<VMSMEType>::kVMSMEIDSize> VMSMEID;
-  typedef ap_uint<VMStubMECMBase<VMSMEType>::kVMSMEBendSize> VMSMEBEND;
-  typedef ap_uint<VMStubMECMBase<VMSMEType>::kVMSMEFinePhiSize> VMSMEFINEPHI;
+  typedef ap_int<VMStubMECMBase<VMSMEType>::kVMSMEBendSize> VMSMEBEND;
+  typedef ap_uint<VMStubMECMBase<VMSMEType>::kVMSMEBendSize - 1> VMSMEBENDPSDISK;
```

Comment:
Why a -1 here?

Reply:
I can't remember the exact cause now, but I use
`auto stubbendReduced=stubdata____.getBendPSDisk();`
in the MEU. I think one of the bend bits carried extra information that I couldn't use when computing the stub index.